### DISCUSSION TITLE
Approval denied 3 times

### DISCUSSION THREAD

**Discussion by rafaelgalvaolm** [community/community#51536] [GitHub Education] [answered]
I am writing this post because i don't know what to do anymore.
I have applied 3 times for the Github pro for students and i have been denied 3 times, but i have used all information available for a student to show i am indeed enrolled in my univesity and my need to use all function of github daily. 
I have used, in the first time, my ISIC card (because it is in english), and it has been denied, after that i have used my internal student id card (it is digital in my university) and for the third and last time i have use a official document from my university (with today's date 03/30/2023) informing that i am ideed enrolled in a Software Engineering major  at my university. What more can i do to prove i am a student at my University.

**Comment by hw88hw88** [comment]
I have been rejected for 5 times for the GitHub pro for students. I am seeking the answers now.

**Reply by rafaelgalvaolm** [reply]
THe weird part is that my friends who go to the same intership as me did with me their application and got accepted, with the same documet, but different uni

**Comment by subeom7** [comment, marked as answer]
Hello @rafaelgalvaolm 

Did you use your school's email to verify? I was able to resolve this issue by using my school's email.

Also, your student ID or any documents you submit should display your expected graduation date. If it does not, ensure that you verify your enrollment using alternative school documents (transcripts, proof of enrollment, Enrollment Certifications, etc.) that confirm your graduation date.

Additionally, for the "How do you plan to use GitHub?" section, make sure to say you are open to contributing to open-source projects.

Lastly, link your institution's webpage: In the "Company" or "Website" section of your profile, consider adding a link to your educational institution's webpage to further validate your academic affiliation.

After making these updates, reapply for the GitHub Student Pack. If you continue to face difficulties, consider submitting the request through GitHub Support directly, as they can provide more specific guidance or investigate any issues with your application:

GitHub Education Support: https://support.github.com/contact/education

Hope this helps!

**Reply by rafaelgalvaolm** [reply]
Thank you for your answer! Well since my uni does not give us an email i will try to get a time of expected graduation doc from my Uni. That is probably the best approach. I, again, thank you.
